src/main/java/io/confluent/connect/hdfs/wal/WALFile.java

```diff
@@ -56,6 +57,8 @@
   private static final int SYNC_HASH_SIZE = 16;   // number of bytes in hash
   private static final int SYNC_SIZE = 4 + SYNC_HASH_SIZE; // escape + hash
 
+  private static final String leaseException = "org.apache.hadoop.hdfs.protocol.AlreadyBeingCreatedException";
```

Comment:
nit: It'd be nice for variable naming to be consistent with our code style for `static final` member fields. 
Thus, it should be something like: `LEASE_EXCEPTION_CLASS_NAME`
(I added more to the name to show it's a class name string and not to actual class. Your call). 

I know we don't run checkstyle currently, but we will soon.